### PR TITLE
feat: Modified hero banner to be skinnable

### PR DESCRIPTION
**Pull request by long-wan-ep**
## Type

* ### Feature
  

## Description

This PR modifies the hero banner on the home page to be skinnable by:
- Adding branded strings that are configurable.
- Adding a banner image that can be changed.
- Styling changes to the hero banner to make text more visible on top of an image.

## Issues

 * https://elasticpath.atlassian.net/browse/DBT-541

## Notes

Screenshots:
1080p
image
ipad
image
iphone X
image

### CODE REVIEW

File: src/strings/BrandedStrings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "front-title": "The Moltin demo store",
+    "front-paragraph-1": "This is a Gatsby demo store, built using the source plugin to fetch products.",
+    "front-paragraph-2": "Product, category and collection pages are statically generated, and all dynamic cart/checkout functionality happens dynamically with our JS SDKs."
+}
```
Comment (shaunmaharaj) [inline src/strings/BrandedStrings.json:5]: Please add new line char

File: src/styles/branding.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+}
+
+.bottom-banner-text {
+  color: #b3b3b3
+}
+
+.bottom-banner-text-bold {
+  color: #ffffff
+}
```
Comment (shaunmaharaj) [inline src/styles/branding.css:46]: Please add new line char

File: src/strings/BrandedStrings.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "front-title": "The Moltin demo store",
```
Comment (shaunmaharaj) [inline src/strings/BrandedStrings.json:2]:
```suggestion
    "front-title": "The Elastic Path Commerce Cloud demo store",
```

**Comment by shaunmaharaj** [comment]
Thanks for this @long-wan-ep! Please rebase and we can merge after comments are addressed 👍

**Comment by long-wan-ep** [comment]
I've rebased and addressed the comments, thanks.

**Pull request merged**